docs/docs/repository.mdx

```diff
@@ -26,21 +29,45 @@ The Repository pattern provides 2 main benefits:
 1. Using the pattern, you can replace your data store without changing your business code.
 2. It encourages you to implement all store operations in one place, making your code more reusable and easy to find.
 
-You can also use the `OOP` approach:
+If working in Angular, you can also use the object-oriented programming (OOP) approach:
```

Comment:
This shouldn't be specific to Angular.

Reply:
👍🏻 I definitely second guessed myself when I wrote that.  I will remove it.

I based my decision to include it on seeing this in the CLI documentation:

<img width="1062" alt="image" src="https://user-images.githubusercontent.com/76440551/168470717-55c9d6ba-2d92-4c7a-9161-3e75c1c2460b.png">